blocks-config/uagb-controls/generateBackgroundCSS.js

```diff
@@ -36,6 +51,11 @@ const generateBackgroundCSS = ( backgroundAttributes, pseudoElementOverlay = {}
 	const yPositionValue = isNaN( yPosition ) || '' === yPosition ? 0 : yPosition;
 	const yPositionTypeValue = undefined !== yPositionType ? yPositionType : '';
 
+	const xPositionOverlayValue = isNaN( xPositionOverlay ) || '' === xPositionOverlay ? 0 : xPositionOverlay;
+	const xPositionOverlayTypeValue = undefined !== xPositionOverlayType ? xPositionOverlayType : '';
+	const yPositionOverlayValue = isNaN( yPositionOverlay ) || '' === yPositionOverlay ? 0 : yPositionOverlay;
```

Comment:
Although this is correct, this can be shortened to remove the check for empty string ( which returns false on `isNaN` )

```suggestion
	const xPositionOverlayValue = ( 'number' !== typeof xPositionOverlay ) ? 0 : xPositionOverlay;
	const xPositionOverlayTypeValue = undefined !== xPositionOverlayType ? xPositionOverlayType : '';
	const yPositionOverlayValue = ( 'number' !== typeof yPositionOverlay ) ? 0 : yPositionOverlay;
```

Reply:
Added changes.